src/categorical_algebra/DataMigrations.jl

```diff
@@ -305,7 +313,13 @@ function (F::SigmaMigration)(d::ACSet; n=100)
       res[domval,h] = codomval
     end
   end
-  return res
+  diagram_map = Dict(map(types(S)) do o
+    s, t= add_srctgt("α_$o")    
+    m = last.(sort(collect(zip([rel_res[x] for x in [s,t]]...))))
+    ff = o ∈ ob(S) ? FinFunction : VarFunction{attrtype_type(D,o)}
+    o => ff(m, nparts(rel_res, nameof(ob_map(F.F⋅i2,o))))
+  end)
+  return res, diagram_map
```

Comment:
@KevinArlin and I had a discussion this morning about how we are going to refactor the types around data migration. We don't need to worry about it for this PR, but just heads up that the API, including return conventions, for the migrations will change. Luckily no except us should depend on sigma migrations since until recently it only worked in a very special case.